mreg_cli/config.py

```diff
@@ -30,6 +30,9 @@
 
 logger = logging.getLogger(__name__)
 
+
+p = platformdirs.PlatformDirs("mreg-cli", "UiO")
```

Comment:
Where is this used?

Reply:
I posted a comment about this at the same time: https://github.com/unioslo/mreg-cli/pull/346#discussion_r1860533626